lib/tasks/build.rake

```diff
@@ -4,6 +4,7 @@ namespace :tailwindcss do
     debug = args.extras.include?("debug")
     verbose = args.extras.include?("verbose")
 
+    Tailwindcss::Engines.bundle
```

Comment:
WDYT of making this a separate rake task? If bundling the engines' CSS is a dependency of both of these tasks, then let's code it that way.